### ISSUE TITLE
Give the metadata to both Watch and Lease

### ISSUE THREAD

**Issue by reptilbud** [mixer/etcd3#46]
I checked your code and you forgot to give the metadata to the Watch client

```javascript
/* rpc#WatchClient */
watch() {
    return this.client.getConnection('Watch').then(cnx => cnx.client.watch());
}
```

Should be

```javascript
/* rpc#WatchClient */
watch() {
    return this.client.getConnection('Watch').then(cnx => cnx.client.watch(cnx.metadata));
}
```

Same things for ```LeaseClient#leaseKeepAlive```

**Comment by reptilbud** [comment]
Will do a PR as soon as the ```npm run build:proto``` command will work

**Comment by connor4312** [comment]
Fixed in 0.2.6.
